packages/core/content-type-builder/admin/src/components/ComponentCard/ComponentIcon/ComponentIcon.js

```diff
@@ -16,20 +16,25 @@ const Wrapper = styled(Flex)`
   }
 `;
 
-export function ComponentIcon({ isActive }) {
+const DefaultIcon = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512">
```

Comment:
Why does this not link to a Icon form the Icon lib?

Reply:
Yeah agree and similar comment above with the CM. Since we are already using it in two packages I really think it makes more sense to just have it one place.